test/e2e/specs/media/settings__media.ts

```diff
@@ -32,56 +37,25 @@ describe( DataHelper.createSuiteTitle( 'Jetpack Settings: Media' ), function ()
 		await testAccount.authenticate( page );
 	} );
 
-	if ( envVariables.JETPACK_TARGET === 'remote-site' ) {
```

Comment:
My understanding is that we're focusing on the Jetpack experience as hosted on dotcom, which is why I felt comfortable removing this clause.

Reply:
Eh, my instinct would be to actually leave this in for now. It does complicate the test a little bit, but the remote, connected site case is something we might want to target in the future, and we have all the support elsewhere for! But if you feel strongly that you want to take it out, that's okay too 🙂

Reply:
I've restored the `remote-site` portion.